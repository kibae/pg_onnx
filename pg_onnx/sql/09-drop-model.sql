SELECT pg_onnx_drop_model('sample_model', '0');
SELECT pg_onnx_drop_model('sample_model', '1');
SELECT pg_onnx_drop_model('sample_model', '2');
