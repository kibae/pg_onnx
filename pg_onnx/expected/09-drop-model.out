SELECT pg_onnx_drop_model('sample_model', '0');
 pg_onnx_drop_model 
--------------------
 t
(1 row)

SELECT pg_onnx_drop_model('sample_model', '1');
 pg_onnx_drop_model 
--------------------
 t
(1 row)

SELECT pg_onnx_drop_model('sample_model', '2');
 pg_onnx_drop_model 
--------------------
 t
(1 row)

